src/reducers/resource.js

```diff
@@ -46,14 +34,44 @@ const actionsWithoutPayload = new Set([
   REFERENCE_CLEAR,
 ]);
 
-function createNewState(state) {
+/**
+ * @param {String} schema
+ * @return {Object}
+ */
+export function createStatus(schema) {
```

Comment:
You should leave it as `createDefaultStatus` or `createInitialStatus` because you are redefining createStatus from ./status. I will fix it. Just so you now that when using in extensions.